cmd/kube-scheduler/app/options/options.go

```diff
@@ -96,13 +97,20 @@ func NewOptions() (*Options, error) {
 			BindPort:    hport,
 			BindAddress: hhost,
 		},
-		Authentication: nil, // TODO: enable with apiserveroptions.NewDelegatingAuthenticationOptions()
-		Authorization:  nil, // TODO: enable with apiserveroptions.NewDelegatingAuthorizationOptions()
+		Authentication: apiserveroptions.NewDelegatingAuthenticationOptions(),
+		Authorization:  apiserveroptions.NewDelegatingAuthorizationOptions(),
 		Deprecated: &DeprecatedOptions{
 			UseLegacyPolicyConfig:    false,
 			PolicyConfigMapNamespace: metav1.NamespaceSystem,
 		},
 	}
+	o.Authentication.RemoteKubeConfigFileOptional = true
+	o.Authorization.RemoteKubeConfigFileOptional = true
+	o.Authorization.AlwaysAllowPaths = []string{"/healthz"}
+
+	o.SecureServing.ServerCert.CertDirectory = "/var/run/kubernetes"
```

Comment:
@sttts is this only used for generation of self-signed certs? can we switch that to in-memory instead to fix https://github.com/kubernetes/kubernetes/issues/68973 and avoid the same issue here? if so, I'm not sure we'd even need CertDirectory

Reply:
opened https://github.com/kubernetes/kubernetes/pull/69884 to enable this. the only change here would be to drop the CertDirectory, leaving the PairName specified.

Reply:
Great to see the in-memory PR :+1: